### PR TITLE
Wrong max level messages

### PR DESCRIPTION
**Pull request by AminArria**

### CODE REVIEW

File: common/locales/en/rebirth.json [hunk 1]
```diff
@@ -16,10 +16,13 @@
...
+    "rebirthAchievement100": "You've begun a new adventure! This is Rebirth <%= number %> for you, and the highest Level you've attained is 100 or higher. To stack this Achievement, begin your next new adventure when you've reached at least 100!",
     "rebirthBegan": "Began a New Adventure",
     "rebirthText": "Began <%= rebirths %> New Adventures",
     "rebirthOrb": "Used an Orb of Rebirth to start over after attaining Level",
+    "rebirthOrb100": "Used an Orb of Rebirth to start over after attaining Level 100 or higher",
     "rebirthPop": "Begin a new character at Level 1 while retaining achievements, collectibles, and tasks with history.",
     "rebirthName": "Orb of Rebirth",
-    "reborn": "Reborn, max level <%= reLevel %>"
+    "reborn": "Reborn, max level <%= reLevel %>",
+    "reborn100": "Reborn, max level 100+",
```
Comment (crookedneighbor) [inline common/locales/en/rebirth.json]: Trailing comma makes the JSON file invalid.

File: website/views/shared/avatar/index.jade [hunk 2]
```diff
@@ -26,5 +26,7 @@ mixin herobox(opts)
       span.glyphicon.glyphicon-circle-arrow-up(tooltip=env.t('buffed'), tooltip-append-to-body="true",
         ng-show='#{buffs}.str || #{buffs}.per || #{buffs}.con || #{buffs}.int || #{buffs}.stealth')
       span(tooltip=env.t('level'), tooltip-append-to-body="true") {{profile.stats.lvl}}
-      span.glyphicon.glyphicon-plus-sign(ng-show='profile.achievements.rebirths',
+      span.glyphicon.glyphicon-plus-sign(ng-if='::profile.achievements.rebirthLevel < 100', ng-show='profile.achievements.rebirths',
         tooltip=env.t('reborn', {reLevel: "{{profile.achievements.rebirthLevel}}"}), tooltip-append-to-body="true")
```
Comment (crookedneighbor) [inline website/views/shared/avatar/index.jade]: I feel like you could do something like:

``` jade
- var rebirthLevel = "{{profile.achievements.rebirthLevel < 100 ? profile.achievements.rebirthLevel : 100+}}"
span.glyphicon.glyphicon-plus-sign(ng-show='profile.achievements.rebirths',
  tooltip=env.t('reborn', {reLevel: rebirthLevel}), tooltip-append-to-body="true")
```

That would eliminate the need for an extra translation string.

Comment (AminArria) [inline website/views/shared/avatar/index.jade]: Wow nice one. Will make the change right now

**Comment by crookedneighbor** [comment]
This will count towards a contributor tier! What's your User ID? You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.

**Comment by AminArria** [comment]
Thanks.
d1c621be-418e-479d-8892-95a962533ca9

**Comment by crookedneighbor** [comment]
Tier awarded! You can read more about [contributor tiers and rewards here](http://habitrpg.wikia.com/wiki/Contributor_Rewards).

Note that as your tier increases, it takes progressively more work to achieve the next one, so for the higher tiers, larger PRs or a few PRs are required. There's some information about that at Contributor Rewards, but feel free to ask here if you have questions!

**Pull request merged**